tests/test.py

```diff
@@ -569,3 +569,97 @@ def test_pass_inner_group_with_statement(self):
                 'itxn_submit'
             ]
         )
+
+
+class TestOperators(unittest.TestCase):
+
+    def test_binary(self):
+        teal = compile_min([
+            'assert(1 || 2)'
+        ])
+        self.assertEqual(teal, [
+            'pushint 1',
+            'pushint 2',
+            '||',
+            'assert',
+        ])
+
+    def test_unary_literal_int(self):
+        teal = compile_min([
+            'assert(!1)'
+        ])
+        self.assertEqual(teal, [
+            'pushint 1',
+            '!',
+            'assert',
+        ])
+
+    def test_unary_literal_bytes(self):
+        teal = compile_min([
+            'assert(b~"\x00\x00\x00\x00\x00\x00\x00\x00")'
+        ])
+        self.assertEqual(teal, [
+            'pushbytes "\x00\x00\x00\x00\x00\x00\x00\x00"',
+            'b~',
+            'assert',
```

Comment:
I appreciate you likely already know and are just checking the translation is correct, but `assert` can only accept uints. Would `assert(btoi(b~"\x00\x00\x00\x00\x00\x00\x00\x00"))` be a better test here?